test/monitoring_console/manager_monitoring_console_test.go

```diff
@@ -937,4 +937,154 @@ var _ = Describe("Monitoring Console test", func() {
+			/*
+				Test Steps
+				1.  Deploy Standalone with name "search-head-adhoc"
+				2.  Wait for Standalone to go to READY
+				3.  Deploy Monitoring Console
+				4.  Wait for Monitoring Console status to be READY
+				5.  Verify Standalone configured in Monitoring Console Config Map
+				6.  Verify Monitoring Console Pod has correct peers in Peer List
+				7.  Deploy 2nd Standalone with name "search-head"
+				8.  Wait for Second Standalone to be READY
+				9.  Wait for Monitoring Console status to go UPDATING then READY
+				10. Verify both Standalone configured in Monitoring Console Config Map
+				11. Verify both Standalone configured in Monitoring Console Pod Peers String
+				12. Delete 2nd Standalone
+				13. Wait for Monitoring Console to go to UPDATING then READY
+				14. Verify only first Standalone configured in Monitoring Console Config Map
+				15. Verify only first Standalone configured in Monitoring Console Pod Peers String
+			*/
+
+			standaloneOneName := "search-head-adhoc"
+			mcName := deployment.GetName()
+			spec := enterpriseApi.StandaloneSpec{
+				CommonSplunkSpec: enterpriseApi.CommonSplunkSpec{
+					Spec: enterpriseApi.Spec{
+						ImagePullPolicy: "IfNotPresent",
+					},
+					Volumes: []corev1.Volume{},
+					MonitoringConsoleRef: corev1.ObjectReference{
+						Name: mcName,
+					},
+				},
+			}
+			standaloneOne, err := deployment.DeployStandaloneWithGivenSpec(ctx, standaloneOneName, spec)
+			Expect(err).To(Succeed(), "Unable to deploy standalone instance")
+
+			// Wait for standaloneOne to be in READY Status
+			testenv.StandaloneReady(ctx, deployment, deployment.GetName(), standaloneOne, testcaseEnvInst)
+
+			// Deploy MC and wait for MC to be READY
+			mc, err := deployment.DeployMonitoringConsole(ctx, deployment.GetName(), "")
+			Expect(err).To(Succeed(), "Unable to deploy Monitoring Console instance")
+
+			// Verify MC is Ready and stays in ready state
+			testenv.VerifyMonitoringConsoleReady(ctx, deployment, deployment.GetName(), mc, testcaseEnvInst)
+
+			// Check standaloneOne is configure in MC Config Map
+			standalonePods := testenv.GeneratePodNameSlice(testenv.StandalonePod, standaloneOneName, 1, false, 0)
+
+			testcaseEnvInst.Log.Info("Checking for Standalone Pod on MC Config Map")
+			testenv.VerifyPodsInMCConfigMap(ctx, deployment, testcaseEnvInst, standalonePods, "SPLUNK_STANDALONE_URL", mcName, true)
+
+			// Check standaloneOne Pod in MC Peer List
+			testcaseEnvInst.Log.Info("Check standalone  instance in MC Peer list")
+			testenv.VerifyPodsInMCConfigString(ctx, deployment, testcaseEnvInst, standalonePods, mcName, true, false)
+
+			// get revision number of the resource
+			resourceVersion := testenv.GetResourceVersion(ctx, deployment, testcaseEnvInst, mc)
+
+			// Check standaloneOne Pod in MC Peer List
+			testcaseEnvInst.Log.Info("Check standalone  instance in MC Peer list")
+			testenv.VerifyPodsInMCConfigString(ctx, deployment, testcaseEnvInst, standalonePods, mcName, true, false)
```

Comment:
Why is this needed again? We have already checked this in lines 994-996

Reply:
Fixed